`git2/sys/mempack.h` fails to include `git2/buffer.h`

Header file `git2/sys/mempack.h` should include `git2/buffer.h`:
~~~
/usr/local/include/git2/sys/mempack.h:67:22: error: unknown type name 'git_buf'
~~~

### Reproduction steps

### Expected behavior

### Actual behavior

### Version of libgit2 (release number or SHA1)
0.26.0

### Operating system(s) tested

--- a/include/git2/sys/mempack.h
+++ b/include/git2/sys/mempack.h
@@ -11,6 +11,7 @@
 #include "git2/types.h"
 #include "git2/oid.h"
 #include "git2/odb.h"
+#include "git2/buffer.h"
 
 /**
  * @file git2/sys/mempack.h
@@ -38,10 +39,10 @@ GIT_BEGIN_DECL
  *	Subsequent reads will also be served from the in-memory store
  *	to ensure consistency, until the memory store is dumped.
  *
- *	@param out Poiter where to store the ODB backend
+ *	@param out Pointer where to store the ODB backend
  *	@return 0 on success; error code otherwise
  */
-int git_mempack_new(git_odb_backend **out);
+GIT_EXTERN(int) git_mempack_new(git_odb_backend **out);
 
 /**
  *	Dump all the queued in-memory writes to a packfile.
@@ -64,7 +65,7 @@ int git_mempack_new(git_odb_backend **out);
  *	@param backend The mempack backend
  *	@return 0 on success; error code otherwise
  */
-int git_mempack_dump(git_buf *pack, git_repository *repo, git_odb_backend *backend);
+GIT_EXTERN(int) git_mempack_dump(git_buf *pack, git_repository *repo, git_odb_backend *backend);
 
 /**
  *	Reset the memory packer by clearing all the queued objects.
@@ -78,7 +79,7 @@ int git_mempack_dump(git_buf *pack, git_repository *repo, git_odb_backend *backe
  *
  *	@param backend The mempack backend
  */
-void git_mempack_reset(git_odb_backend *backend);
+GIT_EXTERN(void) git_mempack_reset(git_odb_backend *backend);
 
 GIT_END_DECL
 

--- a/src/odb_mempack.c
+++ b/src/odb_mempack.c
@@ -9,6 +9,7 @@
 
 #include "git2/object.h"
 #include "git2/sys/odb_backend.h"
+#include "git2/sys/mempack.h"
 #include "fileops.h"
 #include "hash.h"
 #include "odb.h"

